docs/getting-started/comparison.md

```diff
@@ -481,3 +481,9 @@ const Component = () => {
   // ...
 }
 ```
+
+## Npm Downloads Trend
+
+- [Npm Downloads Trend of State Management Libraries for React](https://npm-compare.com/react-redux,jotai,recoil,valtio/#timeRange=THREE_YEARS)
```

Comment:
you don't want to include zustand? also, `@reduxjs/toolkit` is more comparable. what do you think?
```suggestion
- [Npm Downloads Trend of State Management Libraries for React](https://npm-compare.com/@reduxjs/toolkit,zustand,recoil,jotai,valtio/#timeRange=THREE_YEARS)
```

Reply:
My apologies for overlooking the crucial addition of `zustand`. It has now been included in the merge request. Additionally, I agree that `@reduxjs/toolkit` would be a better fit for this document. Thank you for bringing it to my attention.